### DISCUSSION TITLE
What does "Minimum/maximum temperature allowed" do?

### DISCUSSION THREAD

**Discussion by vodkapmp** [jmcollin78/versatile_thermostat#727] [Q&A] [answered]
So in the main configuration the outdoor sensor has two input fields that I can't find any documentation for.

"Minimum temperature allowed" and "Maximum temperature allowed"
What is the purpose of setting these? Does this make vTherm cap out the values below/above these values?

Also are negative values allowed? 
The coldest outdoor temperatures where I live is -46c, and the warmest is +35c. 
Should I be inputting this? I really don't understand what these do, and there's nothing in the documentation about what they do.

**Comment by vodkapmp** [comment, marked as answer]
Now that I think about it, these values might not be relavant to the sensor at all...
These values are for the min/max temperature set for the device being controlled. I assume?

**Reply by jmcollin78** [reply]
Yes exactly !
